src/ServiceProvider.php

```diff
@@ -52,6 +52,7 @@ public function register(Application $app)
         $app['thumbnails.error_image'] = null;
         $app['thumbnails.cache_time'] = null;
         $app['thumbnails.limit_upscaling'] = true;
+        $app['thumbnails.restrict_alias'] = false;
```

Comment:
Maybe `restrict_route`? At first glance restrict_alias didn't communication a flag to restrict the normal route to only defined/named ones.